TreningGenerator/_Model/ExercisesModel.swift

```diff
@@ -5,37 +5,40 @@ import GameplayKit
 struct ExercisesModel {
```

Comment:
In the future you should think about moving it to a `.plist` or aby other `resource` file :) That way you could update the app by simply changing the `string` file.